packages/venia-ui/lib/components/Checkout/checkoutButton.css

```diff
@@ -0,0 +1,4 @@
+.icon {
+    composes: root from '../Icon/icon.css';
+    --stroke: white;
```

Comment:
An alternative would be to update the `button.css` for "high priority" (ie black background) buttons to automatically have a stroke of white for svgs:

```css

.filled svg {
    --stroke: white;
}
```

That's a bit opinionated though so I prefer our standard method of providing overwritable styles.